numpy/core/numeric.py

```diff
@@ -1815,24 +1835,24 @@ def isscalar(element):
     +--------------------------------------+---------------+-------------------+
     | x                                    |``isscalar(x)``|``np.ndim(x) == 0``|
     +======================================+===============+===================+
-    | PEP 3141 numeric objects (including  | ``True``      | ``True``          |
-    | builtins)                            |               |                   |
+    | PEP 3141 numeric objects (including | ``True`` | ``True`` |
+    | builtins)                           |          |          |
     +--------------------------------------+---------------+-------------------+
     | builtin string and buffer objects    | ``True``      | ``True``          |
     +--------------------------------------+---------------+-------------------+
-    | other builtin objects, like          | ``False``     | ``True``          |
-    | `pathlib.Path`, `Exception`,         |               |                   |
-    | the result of `re.compile`           |               |                   |
+    | other builtin objects, like  | ``False`` | ``True`` |
+    | `pathlib.Path`, `Exception`, |           |          |
+    | the result of `re.compile`   |           |          |
     +--------------------------------------+---------------+-------------------+
-    | third-party objects like             | ``False``     | ``True``          |
-    | `matplotlib.figure.Figure`           |               |                   |
+    | third-party objects like   | ``False`` | ``True`` |
+    | `matplotlib.figure.Figure` |           |          |
     +--------------------------------------+---------------+-------------------+
     | zero-dimensional numpy arrays        | ``False``     | ``True``          |
     +--------------------------------------+---------------+-------------------+
     | other numpy arrays                   | ``False``     | ``False``         |
     +--------------------------------------+---------------+-------------------+
-    | `list`, `tuple`, and other sequence  | ``False``     | ``False``         |
-    | objects                              |               |                   |
+    | `list`, `tuple`, and other sequence | ``False`` | ``False`` |
+    | objects                             |           |           |
```

Comment:
The proposed changes here make the table less readable in the docstring (e.g. `np.core.isscalar?` in IPython) so I think the changes to this table should be reverted.

Reply:
In fact, the changes have to be reverted - I get a malformed table error here when trying to build the docs.

Reply:
Sorry about that. I use a plugin to format tables and I didn't notice this one when I was formatting the table.